src/components/Organisms/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import './NavBarStyles.scss';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTicketAlt, faBars } from '@fortawesome/free-solid-svg-icons';
+import SearchBar from '../../Atoms/SearchBar';
+
+function NavBar() {
+  const [collapse, setCollapse] = useState();
+  const [toggleClass, setToggleClass] = useState('');
+  const [toggleAria, setToggleAria] = useState('');
+  const [navClass, setNavClass] = useState('');
+
+  const updateToggleClass = (isActive) => {
+    if (isActive) {
+      setToggleClass('navbar-toggler collapsed');
+      setToggleAria('false');
```

Comment:
Aria is great but I wouldn't go through the extra effort to update something like this. It's mostly for screen readers and unless you need to be compliant with specific standards it can burn A LOT of time.